test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -5216,6 +5228,51 @@ public void testModifiedFolder() throws Exception {
   }
 
   @Test
+  public void testModifiedFolderUpdatedDocumentsQuery() throws Exception {
+    String parentId = FOLDER.pad("FFF1");
+    String parentFolder = "/FFF1";
+    insertFolder(EPOCH_1970, parentId, parentFolder);
+    String folderId = FOLDER.pad("FFF2");
+    String folder = "/FFF1/FFF2";
+    insertFolder(FEB_1970, folderId, folder);
+    insertDocument(FEB_1970, DOCUMENT.pad("aaa"), folder + "/aaa", folderId);
+    insertDocument(MAR_1970, DOCUMENT.pad("bbb"), folder + "/bbb", folderId);
+
+    checkModifiedDocIdsPushed(
+        ImmutableMap.of(),
+        startPaths(folder),
+        new Checkpoint(FEB_1970, DOCUMENT.pad("aaa")),
+        makeExpectedDocIds(folder, folder, "bbb"),
+        new Checkpoint(MAR_1970, DOCUMENT.pad("bbb")));
+
+    StringBuilder query = new StringBuilder();
+    query.append("SELECT i_chronicle_id, r_object_id, object_name, ")
+    .append("DATETOSTRING_LOCAL(r_modify_date, ''yyyy-mm-dd hh:mi:ss'') ")
+    .append("AS r_modify_date_str ")
+    .append("FROM dm_sysobject ")
+    .append("WHERE FOLDER(''/FFF1'',descend) ")
+    .append("AND ((r_modify_date = DATE(''{0}'',''yyyy-mm-dd hh:mi:ss'') ")
+    .append("AND r_object_id > ''{1}'') ")
+    .append("OR (r_modify_date > DATE(''{0}'',''yyyy-mm-dd hh:mi:ss''))) ")
+    .append("UNION ")
+    .append("SELECT i_chronicle_id, r_object_id, object_name, ")
+    .append("DATETOSTRING_LOCAL(r_modify_date, ''yyyy-mm-dd hh:mi:ss'') ")
+    .append("AS r_modify_date_str ")
+    .append("FROM dm_sysobject ")
+    .append("WHERE r_object_id = ''")
+    .append(DOCUMENT.pad("aaa"))
+    .append("'' ")
+    .append("ORDER BY 3 ASC,2 ASC ENABLE(RETURN_TOP 500)");
```

Comment:
Yeah, but it's not germane, and doesn't affect the test or anything else. More cruft just makes is hard to tell what's real and needed, and what's random.